maddymz/irondb
# US628

User story updates database page and adds search fields/options. Body links and data export will be updated separately.

To Test: (be sure to use the new schema, delete pg-data if necessary)
1. cd into repo and checkout to branch US628.
2. Make sure docker is running.
3. Run `npm install`
4. Run `gulp sass`
5. Run `gulp js`
6. Run `docker-compose up`

To test:
1. Check that the link and search boxes from the landing page work.
2. Check that search parameters on the db-page work, best way to check the element search is so select `S` and `minor`, this is exclusive to a single body, so `IN` will result in `Tombigbee` and `NOT` will result in `Guanaco`.

Hidden elements do not effect the search. 

There are some graphical issues in spacing for the search panel, the spacing is dynamically set when an element is shown/hidden, but the results of the search is bugged. These will have to be addressed in the next sprint. Resizing the window is also a bit of an issue.

Expected style:
![screen shot 2019-02-15 at 12 56 38 pm](https://user-images.githubusercontent.com/6512755/52875022-41416f80-3121-11e9-84bc-59ed9917fc23.jpg)
![screen shot 2019-02-15 at 12 56 52 pm](https://user-images.githubusercontent.com/6512755/52875028-43a3c980-3121-11e9-932b-1eaac4fa9c02.jpg)
